### PR TITLE
updates manage page to include links for activate / deactivate, and...

### PR DESCRIPTION
**Pull request by adambray**
fixes corresponding bug in the memberships table caching

### CODE REVIEW

File: app/views/cohorts/show.html.erb [hunk 1]
```diff
@@ -9,14 +9,17 @@
   <div>
     <h4>Roster</h4>
 
-    <% cache cache_key_for_cohort(@cohort, @active_memberships, current_user) do %>
+    <%  # these two tables are both cached using @cohort.memberships rather than
+        # their corresponding subset to handle the edge case where a user is
+        # activated or de-activated %>
+    <% cache cache_key_for_cohort(@cohort, @cohort.memberships, current_user, "active") do %>
```
Comment (jshawl) [inline app/views/cohorts/show.html.erb]: mismatch erb tags from multiline comment?

File: app/views/cohorts/show.html.erb [hunk 2]
```diff
@@ -9,14 +9,17 @@
   <div>
     <h4>Roster</h4>
 
-    <% cache cache_key_for_cohort(@cohort, @active_memberships, current_user) do %>
+    <% # these two tables are both cached using @cohort.memberships rather than %>
+    <% # their corresponding subset to handle the edge case where a user is %>
+    <% # activated or de-activated %>
```
Comment (adambray) [inline app/views/cohorts/show.html.erb]: I'm ok with these changes, but were they strictly necessary? I tested this and the app worked fine for me. In other words, I don't think the ERB tags were mismatched, just multiline, which is allowed in ERB, AFAIK.

Comment (jshawl) [inline app/views/cohorts/show.html.erb]: not necessary - just a fix for syntax highlighting

**Comment by jshawl** [comment]
:+1: good 2 merge

**Comment by andrewsunglaekim** [comment]
I like the table alot better  =D :+1:

**Pull request merged**